src/BulkWriter/Properties/Resources.resx

```diff
@@ -140,4 +159,10 @@
   <data name="PropertyInfoExtensions_PropertyNotDeclaredOnType" xml:space="preserve">
     <value>The supplied property is not declared on a type.</value>
   </data>
+  <data name="EnumerableDataReader_GetBytes_OrdinalDoesNotMapToProperty" xml:space="preserve">
+    <value>The ordinal does not map to property</value>
```

Comment:
Could you add a period to these exception messages? See https://docs.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions#use-grammatically-correct-error-messages

Reply:
Sure. I forgot this in my new PR.